events.go

```diff
@@ -137,6 +137,22 @@ type Event interface {
 // concrete event types.
 type BasicEvent struct{ *js.Object }
 
+type EventOptions struct {
+	Bubbles    bool
+	Cancelable bool
+}
+
+func CreateEvent(typeArg string, opts *EventOptions) *BasicEvent {
```

Comment:
I'd prefer `EventOptions`, it's simpler and makes adding/removing options a smaller diff.